core/domain/services/graphql/mutators/EntityReorder.php

```diff
@@ -49,23 +49,18 @@ public static function mutateAndGetPayload()
                     esc_html__('Sorry, you do not have the required permissions to reorder entities', 'event_espresso')
                 );
             }
-
+            $ok = true;
+            // 'entities', 'entityGuids', 'entity_db_ids', 'entityType', 'keyPrefix'
             $details = EntityReorder::prepareEntityDetailsFromInput($input);
-
             $orderKey  = $details['keyPrefix'] . '_order'; // e.g. "TKT_order"
 
-            $ok = false;
-
             // We do not want to continue reorder if one fails.
             // Thus wrap whole loop in try-catch
             try {
-                foreach ($details['entityDbids'] as $order => $entityDbid) {
-                    $args = [
-                        $orderKey => $order + 1,
-                    ];
-                    $details['entities'][ $entityDbid ]->save($args);
+                foreach ($details['entity_db_ids'] as $order => $entityDbid) {
+                    $args = [ $orderKey => $order + 1 ];
+                    $ok = $details['entities'][ $entityDbid ]->save($args) ? $ok : false;
```

Comment:
this was done to improve feedback, since the previous way would return true even if updates failed